schema/defs.yaml

```diff
@@ -189,33 +200,58 @@ description: Common schema definitions shared by all themes
         language: { "$ref": "#/$defs/propertyDefinitions/language" }
         value:
           type: string
-    names:
+          minLength: 1
+          pattern: ^(\S.*)?\S$    # Leading and trailing whitespace are not allowed.
+    allNames:
```

Comment:
Why extract common names as a separate property? As far as I see, there is no clear benefit from it. Or am I missing something?

Reply:
I found my answer in division.locality_type